video/out/gpu/d3d11_helpers.c

```diff
@@ -283,7 +287,7 @@ bool mp_d3d11_create_present_device(struct mp_log *log,
         goto done;
     }
 
-    adapter = get_d3d11_adapter(log, adapter_name, NULL);
+    adapter = get_d3d11_adapter(log, bstr0(adapter_name), NULL);
 
     if (adapter_name && !adapter) {
         mp_warn(log, "Adapter '%s' was not found in the system! "
```

Comment:
Maybe this should be changed to something along the lines of "Adapter matching '%s' not found in the system", since adapter_name might be a prefix. The message in d3d11_validate_adapter should probably be changed as well (but GitHub won't let me comment on that line?)

Reply:
Good catch, switched.